src/Infrastructure/LeanCode.ClientsUpdates/Contracts/VersionSupport.cs

```diff
@@ -0,0 +1,16 @@
+using LeanCode.Contracts;
+
+namespace LeanCode.ClientsUpdates.Contracts;
+
+public class VersionSupport : IQuery<VersionSupportDTO?>
+{
+    public PlatformDTO Platform { get; set; }
+    public string Version { get; set; } = default!;
+}
+
+public enum VersionSupportDTO
+{
+    UpdateRequired,
+    UpdateSuggested,
+    UpToDate,
```

Comment:
```suggestion
    UpdateRequired = 0,
    UpdateSuggested = 1,
    UpToDate = 2,
```

`enum`s in DTO should always be explicitly numbered, otherwise it is very easy to break backwards compatibility.